task-8 / Добавить возможность создавать в в бд варианты уровня знаний…

coder1coder opened the pull request:
#8

Commit:
task-8 / Добавить возможность создавать в в бд варианты уровня знаний технологии (определив интерфейс) #8

pingvin1308 reviewed (changes requested):
Попробуй так отправить комменты

coder1coder committed:
task-8 / правки по замечаниям

+ переименовал некоторые контракты (название класса не должно быть глаголом)
+ исправил опечатку в интерфейсе ILevelKnowledgeRepository
